### ISSUE TITLE
Can't use backslash in regexp terminal

### ISSUE THREAD

**Issue by SupraSummus** [igordejanovic/parglare#74]
* parglare version: rev ef27e8813455a9b26fb8af71a0d8a7f0d286f793
* Python version: 3.7
* Operating System: linux 4.18.16 x86_64

### Description

I'm unable to load a grammar that has a regexp terminal with backslash (`\`) in it. Grammar is like this:
```
start: "anything";

terminals
t1: /\\/;
t2: /a/;
```

I tried using `t1: /\/;` and also `t1: /\\\\/;` - these didn't work either.

It seems like grammar parser continues to read regexp, as error is raised for line below `t1`. Details are listed below.

### What I Did

```python
from parglare import Grammar

grammar = r"""
start: "anything";

terminals
t1: /\\/;
t2: /a/;
"""

g = Grammar.from_string(grammar)
```
raises
```
Traceback (most recent call last):
  File "test.py", line 11, in <module>
    g = Grammar.from_string(grammar)
  File "/home/jan/programowanie/parglare/parglare/grammar.py", line 1111, in from_string
    return Grammar._parse('parse', grammar_str, **kwargs)
  File "/home/jan/programowanie/parglare/parglare/grammar.py", line 1094, in _parse
    context=context)
  File "/home/jan/programowanie/parglare/parglare/parser.py", line 204, in parse
    raise error
parglare.exceptions.ParseError: Error at 6:5:"\\/;\nt2: /*a/;\n" => Expected: ; or { but found <Name(a)> or <NotComment(a/;)>
```

**Comment by SupraSummus** [comment]
I'm sorry if I'm becoming annoying. I don't mean to be. Parglare is a great parser library - the best one I've seen for python so far. ;) Context is I'm trying to build a dhall (~200 grammar rules) interpreter and during the process I find these small bugs.

**Comment by igordejanovic** [comment]
No, you are not annoying at all. Thanks much for your work! You've found some really interesting cases and issues that, when fixed, will make parglare even better :)

**Comment by igordejanovic** [comment]
Closed via #75
